test/parse-result.js

```diff
@@ -220,6 +220,10 @@ describe('Parse result namespace', () => {
       expect(sourceMaps.first().toValue()).to.deep.equal([[1, 2]]);
     });
 
+    it('should have a convenience method for retrieving source map', () => {
+      expect(element.sourceMap.first().toValue()).to.deep.equal([[1, 2]]);
```

Comment:
Perhaps we can call it `sourceMapValue` and it can return the first source maps value? That way we're keeping it consistent with which methods return elements vs values.